Integrations/integration-symantec-DLP-incident-reportintg-update.yml

```diff
@@ -0,0 +1,221 @@
+description: Integration to fetch and update symantec DLP Incidents
+detaileddescription: Symantec DLP integration uses SOAP API to communicate with Enforce
+  server to fetch and update  incident Data. uses Zeep package for SOAP Api
+configuration:
+- display: 'credentials '
+  name: credentials
+  defaultvalue: ""
+  type: 9
+  required: false
+- display: 'Server URL'
+  name: server-ip
+  defaultvalue: ""
+  type: 0
+  required: false
+- display: Fetch incidents
+  name: isFetch
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Incident type
+  name: incidentType
+  defaultvalue: ""
+  type: 13
+  required: false
+- display: First fetch timestamp (<number> <time unit>, e.g., 12 hours, 7 days, 3     months,
+    1 year)
+  name: fetch_time
+  defaultvalue: 2 months
+  type: 0
+  required: false
+- display: 'Saved Report ID'
+  name: saved-Report-id
+  defaultvalue: ""
+  type: 0
+  required: false
+script:
+  script: |+
+    from requests import Session
+    from zeep import Client
+    from zeep.transports import Transport
+    from requests.auth import AuthBase, HTTPBasicAuth
+    from zeep import helpers
+    import random
+    import datetime
+    import json
+    import requests
+    from requests.packages.urllib3.exceptions import InsecureRequestWarning
+
+    requests.packages.urllib3.disable_warnings()
+
+    # hexdump:732073756468656572206b756d6172
+
+    class SymantecAuth(AuthBase):
+        def __init__(self, USERNAME, PASSWORD, host):
+            self.basic = HTTPBasicAuth(USERNAME, PASSWORD)
+            self.host = host
+
+        def __call__(self, r):
+            if r.url.startswith(self.host):
+                return self.basic(r)
+            else:
+                return r
+
+    SERVER_IP = demisto.params()['server-ip']
+    USERNAME = demisto.params()['credentials']['identifier']
+    PASSWORD = demisto.params()['credentials']['password']
+    SAVED_REPORT_ID = demisto.params()['saved-Report-id']
+    FETCH_TIME = demisto.params()['fetch_time']
+    SEVERITY = {'high':3,'medium':2,'low':1,'info':5}
+
+    wsdl = 'https://{}/ProtectManager/services/v2011/incidents?wsdl'.format(SERVER_IP)
+
+    session = Session()
+    session.auth = SymantecAuth(USERNAME,PASSWORD, "https://{}".format(SERVER_IP))
+    session.verify = False
+    transport = Transport(session=session)
+
+    client = Client(wsdl=wsdl, transport=transport)
+
+    def  get_incident_id(dtobj):
+
+        if type(dtobj) is datetime.datetime:
+            return client.service.incidentList(savedReportId=SAVED_REPORT_ID, incidentCreationDateLaterThan=dtobj)
+        else:
+            return_error('excpected datetime object')
+
+
+
+    def incidentDetail(_id):
+
+        return client.service.incidentDetail(incidentId=_id,includeViolations=True,includeHistory=True)
+        
+    def myconverter(o):
+        if isinstance(o, datetime.datetime):
+            return o.__str__()        
+    
+        
+    def parse_date_range(daterange):
```

Comment:
you didn't have to implement it. You could just use our built in command `parse_date_range`